cves/CVE-2013-3674.yml

```diff
@@ -149,10 +158,10 @@ unit_tested:
     In FFmpeg, they have the FATE suite of regression tests. They are not
     "unit" tests per se, but are automated regression tests. If the fix for the
     vulnerability involved adding or updating a FATE suite, then make fix: true.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: false
```

Comment:
`code_answer` and `fix_answer` should have text that says why the answers are false (saying that there's no tests for this is fine)